### ISSUE TITLE
Error decoding connected accounts

### ISSUE THREAD

**Issue by Buratti** [vapor-community/stripe-kit#130]
It seems like there is a problem decoding the objects returned by `stripe.connectAccounts.retrieve`.

The error I am getting is:

```
valueNotFound(Swift.Double, Swift.DecodingError.Context(codingPath: [_JSONKey(stringValue: "super", intValue: nil)], debugDescription: "Expected Double but found null value instead.", underlyingError: nil)
```

But if I try `curl https://api.stripe.com/v1/accounts/` the returned object is exactly as described in the Stripe documentation.

I tried to investigate the error but I was not able to find any single reference to `super` in the whole stripe-kit project.

**Comment by Andrewangeta** [comment]
@Buratti Are there any references to "super" in the Stripe API docs?

**Comment by Buratti** [comment]
It does not seem so

**Comment by Andrewangeta** [comment]
@Buratti I've seen something similar but having to do with payouts.....
`No value associated with key _JSONKey(stringValue: "super", intValue: nil) ("super"). : 'data -> object'`
This is from a Webhook. Did you ever figure this out or run into this error again?

**Comment by Andrewangeta** [comment]
This was fixed by #138 @Buratti
